app/views/devise/mailer/invitation_instructions.html.erb

```diff
@@ -1,4 +1,4 @@
-<p>Welcome New Sanctuary Coalition!</p>
+<p>Welcome New Sanctuary Coalition (<%= @resource.community.name.upcase %>)!</p>
```

Comment:
Q: Should this say `Volunteer` to match the other file?

Reply:
Yes, thank you!  Sorry for the slow reply!